homeassistant/components/homeassistant/triggers/homeassistant.py

```diff
@@ -3,7 +3,12 @@
 import voluptuous as vol
 
 from homeassistant.const import CONF_EVENT, CONF_PLATFORM
-from homeassistant.core import CALLBACK_TYPE, HassJob, HomeAssistant
+from homeassistant.core import (
+    CALLBACK_TYPE,
+    DOMAIN as CONF_CORE,
```

Comment:
`CONF_` makes it seems like it is an configuration attribute, which this is not.

Reply:
Ah okay. I was triggered by this comment:

https://github.com/home-assistant/core/pull/113844#discussion_r1531655767

`DOMAIN` is imported as `CONF_DOMAIN`.

Reply:
But here it could just be `DOMAIN` I guess.